content/news/006.open-tech-meeting-2024-05.en.md

```diff
@@ -0,0 +1,23 @@
+---
+published: 2024-04-13
+description: presentation of our progress and local tech meetup
+title: Invitation to our Open Tech Meetup
+keywords: [ Tech, Update, Event ]
+authors: [ Maurice, Tassilo ]
+image: 006_open_tech_1.webp
+---
+
+Peering should start soon! That is why we would like to invite you to our Open-Tech Meeting.
```

Comment:
DD-IX is your peering platform. That is why we would like to invite you to our Open Tech Meetup. We want to report about progress we made during the last months, collect your feedback, and discuss future plans with you.